extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -54,6 +54,7 @@ export class LiveListInterface {
    * Update the underlying live list dom structure.
    *
    * @param {?Element} element
+   * @return {time}
```

Comment:
@dvoytenko thanks for correcting. IMO, i think its useful. @cramforce i think you mentioned this sort of aliasing doesn't work with CC right? (correct me if I'm wrong)